demo/Dockerfile

```
@@ -83,6 +83,7 @@ RUN  if [ -z "$WITH_TESTS" ] || [ "$WITH_TESTS" = "false" ]; then \
        WITH_TESTS_ARG=""; \
      fi \
   && conan profile detect \
+  && cd cppagent \
   && conan create . \
```

I believe we could eliminate changing directory by replacing `.` with an absolute path.

```suggestion
  && cd cppagent \
  && conan create /root/agent/cppagent \
```

---

I'll make the change

---

Okay. Now I see that I created wrong suggestion, as I selected only one line instead of two, sorry. :man_facepalming: